app/src/main/res/values/dimen.xml

```diff
@@ -4,6 +4,7 @@
     <dimen name="dp4">4dp</dimen>
     <dimen name="dp7">7dp</dimen>
     <dimen name="dp8">8dp</dimen>
+    <dimen name="dp12">12dp</dimen>
```

Comment:
Well, we do not put some actual sizes of icons to dimens but do have `dp12=12dp`.
Do we need all those dimens? Or am I missing possibility that `dp12` could suddenly become equal to `20dp`?

Reply:
Seems I've put this change at first steps.
I'll roll back this dimen and put directly to layout.
By the way, I propose to remove all this dimension at all, it doesn't make sense with this kind of naming @yunikkk @mapbox/vision-android WDYT?

Reply:
removed values with "dp*" names